lib/matplotlib/axes/_axes.py

```diff
@@ -3277,13 +3277,10 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             # Remove alpha=0 color that _process_plot_format returns
             fmt_style_kwargs.pop('color')
 
-        if ('color' in kwargs or 'color' in fmt_style_kwargs):
-            base_style = {}
-            if 'color' in kwargs:
-                base_style['color'] = kwargs.pop('color')
-        else:
-            base_style = next(self._get_lines.prop_cycler)
-
+        base_style = self._get_lines._getdefaults(
+            set(), {**fmt_style_kwargs, **kwargs})
```

Comment:
Is this where the property cycler gets implicitly advanced?

Reply:
Yes, I don't really like the private API of a private member, but that's how it is right now.